### ISSUE TITLE
Clean up code

### ISSUE THREAD

**Issue by Matriks404** [Matriks404/mlc#20]
The code is not really great right now, also need to check accessibility stuff if it's working properly after next couple of changes (I mean just the high contrast stuff and accessibility-friendly markup because there's not much else anyway).

**Comment by Matriks404** [comment]
Tasks to do for CSS cleanup (ideally should be its own issue):
* [ ] Reorganize SCSS into distinct files, each with its own purpose.
* [ ] Probably get rid of flexbox, since even on IE10/IE11 it gives headaches and it's not really needed anyway in most places. Not sure what to do with block/item list though.
